payments/utils.py

```diff
@@ -160,3 +163,56 @@ def generate_order_number() -> str:
     t = time.time() * 1000000 * random.uniform(1, 1000)
     b = base64.b32encode(struct.pack(">Q", int(t)).lstrip(b"\x00")).strip(b"=").lower()
     return b.decode("utf8")
+
+
+def get_talpa_product_id(
+    product_id: Union[UUID, str],
+    area: Optional[Union[Harbor, WinterStorageArea]] = None,
+):
+    """
+    The required ID for Talpa should have the following format:
+    Tilino_Tulosyksikkö_Sisäinentilaus_Projekti_Toimintoalue_ALV-koodi_Vapaaehtoinenomatuotenumero
+
+    Tilinumero                      Account number
+    Tulosyksikkö                    Business unit
+    Sisäinen Tilaus                 Internal Order
+    Projekti                        Project
+    Toimintoalue                    Function area
+    ALV-koodi                       VAT code
+    Vapaaehtoinen Oma Tuotenumero   Optional own product number
+    """
+    account_number = "340100"
+    project = " "  # Empty
+    vat_code = "44"  # Fixed
+    own_product_number = str(product_id)
+    business_unit = " "
+    function_area = " "
+    internal_order = " "
+
+    if isinstance(area, Harbor):
+        function_area = "292015"
+    if isinstance(area, WinterStorageArea):
+        function_area = "292014"
+
+    if area:
+        if area.region == AreaRegion.EAST:
+            business_unit = "2923301"
+        elif area.region == AreaRegion.WEST:
+            business_unit = "2923302"
+
+        if area.region == AreaRegion.EAST:
+            internal_order = "2923301100"
+        elif area.region == AreaRegion.WEST:
+            internal_order = "2923302200"
```

Comment:
```suggestion
```suggestion
if area.region == AreaRegion.EAST:
    business_unit = "2923301"
    internal_order = "2923301100"
elif area.region == AreaRegion.WEST:
   business_unit = "2923302"
   internal_order = "2923302200"
```
```

Reply:
:derp: I don't know how I didn't see this before